docs/.vuepress/config.ts

```diff
@@ -47,7 +47,22 @@ export default defineUserConfig<DefaultThemeOptions>({
       },
     ],
     ['meta', { name: 'msapplication-TileColor', content: '#3eaf7c' }],
-    ['meta', { name: 'theme-color', content: '#3eaf7c' }],
+    [
+      'meta',
+      {
+        name: 'theme-color',
+        content: '#3eaf7c',
+        media: '(prefers-color-scheme: light)',
+      },
+    ],
+    [
+      'meta',
+      {
+        name: 'theme-color',
+        content: '#22272e',
+        media: '(prefers-color-scheme: dark)',
+      },
+    ],
```

Comment:
Let's revert this change, too. "Multiple theme-color tag" is not coupled with current PR.

Reply:
Okey, reverted in https://github.com/vuepress/vuepress-next/pull/218/commits/3dc1094fc698bb9e8758abd50bd0e3433c4df7fa